package-lock.json

```diff
@@ -4527,6 +4527,11 @@
       "resolved": "https://registry.npmjs.org/@reach/observe-rect/-/observe-rect-1.0.5.tgz",
       "integrity": "sha512-EQLHhquzwAh2TFtMJU0PXJRmhpOrAhzApgoI4Lwn72J7ygrMk3Ys3a3i8KDnV0TetbwAxn4XSHi4pE8sPWUYLg=="
     },
+    "@scarf/scarf": {
+      "version": "1.0.5",
+      "resolved": "https://registry.npmjs.org/@scarf/scarf/-/scarf-1.0.5.tgz",
+      "integrity": "sha512-9WKaGVpQH905Aqkk+BczFEeLQxS07rl04afFRPUG9IcSlOwmo5EVVuuNu0d4M9LMYucObvK0LoAe+5HfMW2QhQ=="
+    },
```

Comment:
I was curious what this new dependency final-form added was, and it turns out to be some [analytics package](https://final-form.org/docs/react-final-form/getting-started#installation).

Maybe we should opt-out from this in our package.json? While I can understand the developer wanting to get some analytics about where his package is installed, and it would only affect developers (since our end users don't install any JS from npm), I usually don't like analytics that much (unless I see the results ;)), so I'd probably [opt-out](https://github.com/scarf-sh/scarf-js#as-a-user-of-a-package-using-scarf-how-can-i-opt-out-of-analytics) from them.